app/Http/Controllers/Order/ExtendedOrderController.php

```diff
@@ -55,7 +57,7 @@ private function getBaseQueryForOrders()
             ->leftJoin("products", 'orders.product', "=","products.id")
             ->select(
                 'orders.id', 'orders.created_at', 'price_override', 'order_status', 'product', 'number', 'serial_key',
-                'subscriptions.update_ends_at as subscription_ends_at', "subscriptions.id as subscription_id", "subscriptions.version as product_version",
+                'subscriptions.update_ends_at as subscription_ends_at', "subscriptions.id as subscription_id", "subscriptions.version as product_version","subscriptions.updated_at as updated_at", 
```

Comment:
no need to write `as updated_at`. It will be same as column name

Reply:
Fixed